lib/datasets/rdatasets.rb

```diff
@@ -0,0 +1,126 @@
+require_relative "dataset"
+require_relative "tar_gz_readable"
+
+module Datasets
+  class RDatasets < Dataset
+    class FileNotFound < StandardError; end
+
+    class Master < Dataset
+      include TarGzReadable
+
+      def initialize
+        super
+        @metadata.id = "RDatasets"
+        @metadata.name = "RDatasets"
+        @metadata.url = "https://vincentarelbundock.github.io/Rdatasets/"
+        @metadata.licenses = ["GPL-3"]
+        @data_url = "https://github.com/vincentarelbundock/Rdatasets/archive/refs/heads/master.tar.gz"
```

Comment:
How about just reading from `https://raw.githubusercontent.com/vincentarelbundock/Rdatasets/master/datasets.csv`?

Reply:
This tarball contains not only `datasets.csv`, the catalog of Rdatasets, but also all the datasets in Rdatasets.
Do you think it is better that we download each dataset separately on demand?

Reply:
Yes. I think so.
Users can download only datasets they need.

What do you think about this?

Reply:
The only thing I suspect is that whether we should manage every dataset at once.

Reply:
@kou I've rewritten the dataset so that each dataset downloads only a csv file for itself. https://github.com/red-data-tools/red-datasets/pull/99/commits/655035ff5d316995e8f67a553e7fb7ad8d4baf7c